core-bundle/src/Resources/contao/classes/BackendTemplate.php

```diff
@@ -44,6 +46,19 @@ public function parse()
 		return $strBuffer;
 	}
 
+	/**
+	 * Return a response object
+	 *
+	 * @return Response The response object
+	 */
+	public function getResponse()
+	{
+		$response = parent::getResponse();
+		$response->headers->set('Cache-Control', 'no-cache, no-store');
+
+		return $response->setPrivate();
```

Comment:
why did you add `setPrivate`? The response should **not** be privately cacheable by the browser?